CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
 # io.js ChangeLog
 
+## 2015-05-07, Version 2.0.1, @rvagg
+
+### Notable changes
+
+* **async_wrap**: it is now possible to filter by providers (Trevor Norris) [#1614](https://github.com/iojs/io.js/pull/1614)
+* **libuv**: resolves numerous io.js issues:
+  - [#862](https://github.com/iojs/io.js/issues/862) prevent spawning child processes with invalid stdio file descriptors
+  - [#1397](https://github.com/iojs/io.js/issues/1397) fix EPERM error with fs.access(W_OK) on Windows
+  - [#1621](https://github.com/iojs/io.js/issues/1621) build errors associated with the bundled libuv
+  - [#1512](https://github.com/iojs/io.js/issues/1512) should properly fix Windows termination errors
+* **addons**: the `NODE_DEPRECATED` macro was causing problems with compiling addons with older compilers, this should now be resolved (Ben Noordhuis) [#1626](https://github.com/iojs/io.js/pull/1626)
+* **V8**: upgrade V8 from 4.2.77.18 to 4.2.77.20 with minor changes
+
+### Known issues
+
+See https://github.com/iojs/io.js/labels/confirmed-bug for complete and current list of known issues.
```

Comment:
hmmm, I didn't really pay attention to the fact that renaming labels may break links. Noted.